src/reader.rs

```diff
@@ -3098,17 +3107,69 @@ impl<'a> Reader<'a> {
                 };
                 let (elt, _el) = self.active_edit_line();
                 if let Some(target) = self.function_pop_arg() {
-                    let success = self.jump(direction, precision, elt, target);
+                    let success =
+                        self.jump_and_remember_last_jump(direction, precision, elt, target);
 
                     self.input_data.function_set_status(success);
                 }
             }
+            rl::JumpToMatchingBracket | rl::JumpTillMatchingBracket => {
+                let (elt, _el) = self.active_edit_line();
+                let el = self.edit_line(elt);
+                let l_brackets = vec!['(', '[', '{'];
+                let r_brackets = vec![')', ']', '}'];
+                let position = el.position();
+                let cur_char = match c {
```

Comment:
Alternative variable name: `jump_from`. This way `targets` can be renamed to `jump_to` to make the code symmetrical

Reply:
Applied the suggestion